go.mod

```diff
@@ -42,25 +42,25 @@ require (
 	gopkg.in/square/go-jose.v2 v2.6.0
 	gopkg.in/yaml.v2 v2.4.0
 	gopkg.in/yaml.v3 v3.0.1
-	k8s.io/api v0.24.2
-	k8s.io/apimachinery v0.24.3
-	k8s.io/apiserver v0.24.2
-	k8s.io/client-go v0.24.2
+	k8s.io/api v0.25.4
```

Comment:
Why all the go mod changes?

Reply:
I bumped `controller-runtime` to use their latest logger attributes (`reconcileID` for example).

Also, go upgraded some modules when adding `component-base`.